cmd/minishift/cmd/oc_env.go

```diff
@@ -96,3 +108,16 @@ func init() {
 	RootCmd.AddCommand(ocEnvCmd)
 	ocEnvCmd.Flags().StringVar(&forceShell, "shell", "", "Force setting the environment for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh]. Default is auto-detect.")
 }
+
+// Get the oc path as per the current profile.
+// InstanceConfig.OcPath is set in minishift start or profile set. So when oc-env is called with --profile
+// it points to last minishift start or profile set.
+func getOcPath() string {
```

Comment:
I think something similar is done in `start.go` or `clusterup.go`? To me, this feels not specific to this command only. At least seems like a utility function...

Reply:
@gbraad In start.go/clusterup.go this is done as part of the start process. This is a special case for oc-env as `profile set` also takes care of this when changing the oc context. Other commands does not care much about the actual oc binary as long as oc binary's context is set properly. But for oc-env it is cosmetic as we do not want to change the context when user runs the command with --profile i.e. `minishift oc-env --profie profile_name`. Also I have thought about how the code can be removed easily if we want in future.